index.bs

```diff
@@ -95,13 +95,21 @@ Examples {#examples}
 --------------------
 
 ```js
+let userControlledInput = "&lt;img src=x onerror=alert(1)//&gt;";
+
+// Create a DocumentFragment from unsanitized input:
+let s = new Sanitizer();
+let sanitizedFragment = s.sanitize(userControlledInput);
+
+// Replace an element's content from unsanitized input:
+element.innerText = "";
+element.appendChild(s.sanitize(userControlledInput));
+
+// Use the sanitizer to handle additional input that might be undesired for
+// a specific application, for example hyperlinks or plugins.
 let s = new Sanitizer({
-  allowElements: ['a', 'b', ...],
-  allowAttributes: ['c', 'd', 'e', ...],
-  ...
+  blockElements: ['a', 'object', ...],
```

Comment:
Not sure how we agree on how options are currently passed. Maybe leave this one out and use `new Sanitizer(...)` for now?

Reply:
Done.